app/assets/javascripts/discourse/ember-cli-build.js

```diff
@@ -173,6 +180,22 @@ module.exports = function (defaults) {
               exportsPresence: "error",
             },
           },
+          rules: [
+            {
+              test: require.resolve("bootstrap/js/modal"),
+              use: [
+                {
+                  loader: "imports-loader",
```

Comment:
`imports-loader` is otherwise unused as of #23971, so if we don't end up merging this PR, we can remove that dependency